dispatch/static/manager/src/styles/components/event_audit.scss

```diff
@@ -73,3 +73,8 @@
     margin-right: 0.75em;
   }
 }
+
+.pt-button {
```

Comment:
Adding a style to `.pt-button` here will affect all pt-button in the application.
I'm not sure that is what you intended.
Is there a way to just select the one you added?

I think this change in https://github.com/ubyssey/dispatch/commit/8337541fdbd550bc05908ed5aa1f13e960e8b4d9
should cover it, so try merging that in instead.